src/actions/LoginActions.tsx

```diff
@@ -303,7 +306,8 @@ async function createCustomWallets(account: EdgeAccount, fiatCurrencyCode: strin
 
   // Actually create the wallets:
   const options = [...optionsMap.values()]
-  const results = await runWithTimeout(account.createCurrencyWallets(options), 20000, new Error(lstrings.error_creating_wallets)).catch(error => {
+  const timeoutMs = Math.min(options.length * PER_WALLET_TIMEOUT, MIN_CREATE_WALLET_TIMEOUT)
```

Comment:
Use `Math.max` to make sure to pick `MIN_CREATE_WALLET_TIMEOUT` over the first expression if the expression results in a lower value.